docs/next.config.js

```diff
@@ -100,33 +100,8 @@ module.exports = {
           // transpile 3rd party packages with dependencies in this repository
           {
             test: /\.(js|mjs|jsx)$/,
-            include: /node_modules(\/|\\)(material-table|notistack|@material-ui(\/|\\)pickers)/,
-            use: {
```

Comment:
Why remove these?

Reply:
Is have replaced it with the default Babel loader. It's not required per say for the changes of the pull request. Happy to revert to only propose them once we have a live demo.

Reply:
But why? We shouldn't fully transpile these modules since that would hide potential mismatches in transpilation targets. If external libraries would start transpiling for modern browsers by default we wouldn't ever notice. 

Right now it would just add more transpilation work.

Reply:
The change was reverted.

Reply:
We will be able to resume the discussion with a future pull request to add a live demo.

Reply:
> The change was reverted. I have removed the live demo, it wasn't ready.

Sure. Just wanted to make sure we understand each other because I asked for a "why" and you gave me a "what".

Reply:
For the "why", I do no longer have it in context. The build was failing, I copy and paste the configuration of the workspace.